blocks/gnav/gnav.js

```diff
@@ -82,9 +82,7 @@ class Gnav {
     const brand = brandBlock.querySelector('a');
 
     const { className } = brandBlock;
-    const classNameClipped = className.slice(0, -1);
-    const classNames = classNameClipped.split('--');
-    brand.className = classNames.join(' ');
+    brand.className = className;
```

Comment:
@rhudea , this `gnav` block was copied over from the `blog` repo and I see that the change you propose here isn't done there. It still has a [code like this](https://github.com/adobe/blog/blob/9675eadc2037547ecd0460e0f18b23cb1fb6079e/blocks/gnav/gnav.js#L92-L94). So I am not really sue if we require this change for `v7` upgrade. If yes, should this also be done for the `blog` site? Could you please add you inputs?

cc @rofe @kptdobe